app/helpers/pagination_helper.rb

```diff
@@ -0,0 +1,65 @@
+module PaginationHelper
+  include Pagy::Frontend
+
+  # change the default link renderer for will_paginate
+  def will_paginate(collection_or_options = nil, options = {})
+    if collection_or_options.is_a? Hash
+      options = collection_or_options
+      collection_or_options = nil
+    end
+    unless options[:renderer]
+      options = options.merge renderer: PaginationListLinkRenderer
+    end
+    super(*[collection_or_options, options].compact)
+  end
+
+  # Cf https://github.com/ddnexus/pagy/blob/master/gem/lib/pagy/frontend.rb
+  def pagy_nav(pagy, id: nil, aria_label: nil, **vars)
+    return nil unless pagy
+
+    # Keep will_paginate behavior of showing nothing if only one page
+    return nil if pagy.series.length <= 1
+
+    id = %( id="#{id}") if id
+    a  = pagy_anchor(pagy, **vars)
+
+    html = %(<h4 class="landmark heading">#{t('pagy.h4')}</h4>)
```

Comment:
Are invisible h* for navigation purpose still a thing nowadays? Or should we get rid of it?

Reply:
They are for screen readers, we can keep it. You can use the existing key `t("a11y.navigation")` instead of adding a new `pagy` one.